dot/parachain/overseer/mocks_generate_test.go

```diff
@@ -4,4 +4,5 @@
 package overseer
 
 //go:generate mockgen -destination=mocks_test.go -package=$GOPACKAGE . BlockState,Overseer
-//go:generate mockgen -destination=mock_runtime_instance_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/parachain/runtime RuntimeInstance
+//go:generate mockgen -destination=mock_parachain_instance_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/parachain/runtime RuntimeInstance
+//go:generate mockgen -destination=mock_runtime_instance_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/lib/runtime Instance
```

Comment:
This will create mock for all the methods in runtime, many of them we don't need.